contracts/proxy/utils/Initializable.sol

```diff
@@ -83,10 +98,9 @@ abstract contract Initializable {
      */
     modifier initializer() {
         bool isTopLevelCall = !_initializing;
-        require(
-            (isTopLevelCall && _initialized < 1) || (address(this).code.length == 0 && _initialized == 1),
-            "Initializable: contract is already initialized"
-        );
+        if ((!isTopLevelCall || _initialized >= 1) && (address(this).code.length > 0 || _initialized != 1)) {
```

Comment:
I agree in keeping this because you have a better readable sense of this. I just made equivalence with Morgan's Law.

```
// Original
// (isTopLevelCall && _initialized < 1) || (address(this).code.length == 0 && _initialized == 1)
// !(A || B) == !A && !B
// !(isTopLevelCall && _initialized < 1) && !(address(this).code.length == 0 && _initialized == 1)
// !(A && B) == !A || !B
// (!isTopLevelCall || _initialized >= 1) && (address(this).code.length > 0 || _initialized != 1)
```